sphinx/screens/chats/chat-common/chat-common/src/main/java/chat/sphinx/chat_common/ui/ChatViewModel.kt

```diff
@@ -657,6 +647,22 @@ abstract class ChatViewModel<ARGS: NavArgs>(
         }
     }
 
+    private inner class AttachmentFullscreenStateContainer: ViewStateContainer<AttachmentFullscreenViewState>(
+        AttachmentFullscreenViewState.Idle)
```

Comment:
unnecessary as there are no overrides to the `ViewStateContainer`'s open methods/values and it can be used directly.